src/components/procedure/_platform-add-repo.md

```diff
@@ -13,5 +13,10 @@ To add a repository from GitHub, GitLab, or BitBucket, follow these steps:
 4. Click **Scan new project**, and then click **Run scan in CI**.
 5. Select which CI provider for Semgrep to integrate with.
 6. Follow the instructions displayed on Semgrep Cloud Platform page for your particular CI provider.
+7. Optional: For GitHub Actions users, if you have given Semgrep access to more than 1 repository, you can bulk add many repositories.
 
 You have now added a repository to Semgrep Cloud Platform. A scan begins automatically after adding a new repository.
+
+:::caution
+For GitHub users: If you are unable to add repositories, check your [branch protection](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches) settings.
```

Comment:
For CSE reviewer: What does the error message look like here? Is this the right placement for the callout?

Reply:
<img width="610" alt="Screen Shot 2023-06-28 at 4 00 49 PM" src="https://github.com/returntocorp/semgrep-docs/assets/117124494/42bd4c55-70e6-4b18-8948-9e396786d784">


They'll see an error in SCP telling them that they can't commit due to branch protection rules.

I almost think it's more honest/accurate to say something like:
This feature works by Semgrep Cloud Platform commiting a Github Actions workflow file directly into your default branch.  Branch Protection will prevent our ability to bulk onboard this repo.  < if there were docs on zero config scanning, i'd point to them here as an alternative for bulk onboarding >

Reply:
@r2c-david sorry for the late reply, this PR has been updated with your notes resolved! Please let me know if it works for you.